content/kit/pi/_index.md

```diff
@@ -14,3 +14,13 @@ The software included on the Raspberry Pi allows it to run your robot. No graphi
 {{% notice warning %}}
 Please do not attempt to modify the software on the Raspberry Pi. It has been configured and tested in its current configuration. Unauthorized modifications may damage your robot or prevent accurate debugging.
 {{% /notice %}}
+
+## Updating your Pi
+Keeping your kit up to date is very important. It enables us to deploy new features, as well as fix bugs. 
+
+There are 2 ways of updating your kit:
+
+- [Using the `update.tar.xz` file](update-file)
```

Comment:
There is likely more than one update, so this should probably be "an" rather than "the".